pkg/istio-agent/health/health_probers.go

```diff
@@ -175,3 +174,32 @@ func (e *ExecProber) Probe(timeout time.Duration) (ProbeResult, error) {
 	}
 	return Healthy, nil
 }
+
+type EnvoyProber struct {
+	Config *ready.Probe
+}
+
+var _ Prober = &EnvoyProber{}
+
+func (a EnvoyProber) Probe(time.Duration) (ProbeResult, error) {
+	if err := a.Config.Check(); err != nil {
+		return Unhealthy, err
+	}
+	return Healthy, nil
+}
+
+type AggregateProber struct {
```

Comment:
why do we have an `AggregateProber`? When would we need to probe anything besides envoy and the application?